src/config/locales/en/history.js

```diff
@@ -54,12 +56,12 @@ export const history = {
     residence: 'Places you lived',
     employment: 'Employment history',
     education: 'Schools & diplomas',
-    federal: 'Former federal service'
+    federal: 'Former federal service',
   },
   residence: {
     title: 'Where you have lived',
     info:
-      'List the places where you have lived beginning with your present residence and working back 10 years.',
+      'List the places where you have lived beginning with your present residence and working back {{years}} years.',
```

Comment:
Calling this out as the localization string that uses an interpolated variable

Reply:
`i18next` will log a warning, and the variable will be replaced with an empty string (so in this example, effectively "...and working back  years."). It does seem that you can set some global defaults, which we could use, although I've been err-ing on the side of setting defaults as props in the components that are using interpolation.